inc/languages/english/admin/style_themes.lang.php

```diff
@@ -188,7 +188,7 @@
 $l['error_missing_stylesheet_extension'] = "This stylesheet must end with the correct file extension, for example, {1}<em>.css</em>";
 $l['error_invalid_parent_theme'] = "The selected parent theme does not exist. Please select a valid parent theme.";
 $l['error_invalid_templateset'] = "The selected template set does not exist. Please select a valid template set.";
-$l['error_invalid_color'] = "The defined color set \"{1}\" is either having invalid character(s) or not in prescribed format. Only alpha-numeric characters are allowed for naming separated by '='.";
+$l['error_invalid_color'] = "The defined color set \"{1}\" is either having invalid character(s) or not in prescribed format. Only numbers and unicode characters are allowed for naming separated by '='.";
```

Comment:
```suggestion
$l['error_invalid_color'] = "The defined color set \"{1}\" is either having invalid character(s) or not in prescribed format. Only numbers and unicode letters are allowed for naming separated by '='.";
```

Reply:
One other nitpick: "is [...] having" is unnatural syntax to a native English speaker. More natural would be "has". So, I'd suggest the first sentence be reworded to:

"The defined color set \"{1}\" either has invalid character(s) or is not in the prescribed format."

Perhaps, too, a comma after "naming" in the second sentence would be helpful for breaking the two phrases, plus a little more description, such as this:

"Only numbers and Unicode letters are allowed for a color's name and its value, which must be separated by '='."